### PR TITLE
17th Bug Crusade

### PR DESCRIPTION
**Pull request by zumbak04**
Changelog:
- Fixed issue where having dragon culture broke merchant republics.
- "Turn Undead" and "Create Monstrous Undead" decisions no longer require Mana if you have "Turn Undead" quest of the Cult of the Damned.
- Moved capital of the Steamwheedle Cartel to coast province, so the Cartel no longer has chance to switch government from Merchant Republic after elections.
- Kaldorei tributaries no longer can declare war against each others.
- LT.21000 (event where your troops sack province) no longer can fire after you siege Death God, Legion, Shath'yar province.
- Fixed issue where secret members of the Shadow Council could get visible Shadow Councilors in their councils which made them... less secret.
- Fixed issue where secret society members didn't get Under Suspicious modifiers.

Secret changelog:
- Fixed issue where no one could resist the Plague and declare war to the Lich King.
- Removed nomadic_opinion from nomadic_governments.txt since it ain't working.
- Fixed a few unit model positions in North Lordaeron, Quel'thalas and South Khaz Modan.
- Fixed error characterhistory.cpp:545 in error.log
- Fixed error characterhistory.cpp:1270 in error.log
- Removed non-adapted vanilla events after you picked Loa.

### CODE REVIEW

File: map/positions.txt [hunk 1]
```diff
@@ -2493,35 +2493,35 @@
...
 	}
 #Hatchet Hills
 	359=
 	{
-		position={2899.000 1628.000 2903.000 1632.000 2892.000 1622.000 2900.000 1623.000 2905.000 1628.000 0.000 0.000 0.000 0.000}
+		position={2899.000 1628.000 2900.000 1627.000 2892.000 1622.000 2900.000 1623.000 2905.000 1628.000 0.000 0.000 0.000 0.000}
 		rotation={0.000 0.000 0.000 0.000 0.000 0.000 0.000}
 		height={0.000 0.000 0.000 20.000 0.000 0.000 0.000}
 	}
 #Amani Pass
```
Comment (MemeWizard) [inline map/positions.txt]: Mb make it closer to lake?

<details>
<summary>Click to expand</summary>

image

</details>

**Comment by MemeWizard** [comment]
> Fixed error characterhistory.cpp:379 (Reference to undefined dynasty) from error.log

>[characterhistory.cpp:379]: Reference to undefined dynasty.  file: history/characters/240000_qiraji.txt line: 4
[characterhistory.cpp:379]: Reference to undefined dynasty.  file: history/characters/240000_qiraji.txt line: 19
[characterhistory.cpp:379]: Reference to undefined dynasty.  file: history/characters/240000_qiraji.txt line: 34

Hmmm
<details>
<summary>Click to expand</summary>

image
image

</details>

**Comment by MemeWizard** [comment]
> "Turn Undead" and "Create Monstrous Undead" decisions no longer require Mana if you have "Turn Undead" quest of the Cult of the Damned.

You can complete this quest without turning character undead

<details>
<summary>Click to expand</summary>

image

</details>

File: map/positions.txt (hunk 1 above)
Comment (zumbak04) [reply map/positions.txt]: Done!

**Comment by zumbak04** [comment]
> > Fixed issue where Kaldorei tributaries could declare war against each other, but they immediately ended.
> 
> Is invalid rebellion wars related to this fix?

No, it's not related, I'm on the master.
<details>
<summary>Click to expand</summary>

image

</details>

> > Fixed error characterhistory.cpp:379 (Reference to undefined dynasty) from error.log
> 
> > [characterhistory.cpp:379]: Reference to undefined dynasty.  file: history/characters/240000_qiraji.txt line: 4
> > [characterhistory.cpp:379]: Reference to undefined dynasty.  file: history/characters/240000_qiraji.txt line: 19
> > [characterhistory.cpp:379]: Reference to undefined dynasty.  file: history/characters/240000_qiraji.txt line: 34
> 
> Hmmm

Fixed!

> > "Turn Undead" and "Create Monstrous Undead" decisions no longer require Mana if you have "Turn Undead" quest of the Cult of the Damned.
> 
> You can complete this quest without turning character undead

Report as separate issue.

**Comment by geffe1** [comment]
ck2_384
Can't even make them declare war, tried to declare war on Malfy a Chadarius. Is that due to religion? Or am I just being a complete retard?

**Comment by zumbak04** [comment]
> In the screenshot above- I am a vassal of Tyrande, just using it to show how it looked like when I played as Chadarius and tried to declare war on Malfy- even with claims.
> 
> Can't even make them declare war, is that due to religion? Or am I just being a complete retard?

As it should be.

Changed the note from...

> Fixed issue where Kaldorei tributaries could declare war against each other, but they immediately ended.

...to..

> Kaldorei tributaries no longer can declare war against each others.

Because... ye, the note pointed out the problem, but didn't mention how it was fixed.

**Pull request merged**